### PR TITLE
Adds Dracthyr race, Devastation and Preservation spells

### PR DESCRIPTION
**Pull request by joshpc**
# What this does
Adds the library of new abilities for Evoker added in the 10.0 pre-patch.

_Disclaimer:_ I mainly used Wowhead for the CD times and caught a few mistakes already. I believe I caught them all but a second set of eyes would be good.

## Testing Done
- Copied it directly into my version of GladiusEX and seems to be mostly okay. The only issue I've noticed is that the Dracthyr icon (under racials) does not show up.

### CODE REVIEW

File: cooldowns_evoker.lua [hunk 1]
```diff
@@ -0,0 +1,390 @@
...
+
+local SPEC_EVOKER_DEVASTATION = 1467
+local SPEC_EVOKER_PRESERVATION = 1468
+
+-- Evoker/Baseline
+-- Fire Breath
+LCT_SpellData[357208] = {
+  class = "EVOKER",
+  cooldown = 30,
+  -- TODO: Fire Breath's CD is reduced when consuming essence bursts when
```
Comment (vendethiel) [inline cooldowns_evoker.lua:11]: I can't wait having to try to track that..:-)

File: cooldowns_evoker.lua [hunk 2]
```diff
@@ -0,0 +1,390 @@
...
+}
+
+-- Disintegrate
+LCT_SpellData[356995] = {
+  class = "EVOKER"
+}
+
+-- Azure Strike
+LCT_SpellData[362969] = {
+  class = "EVOKER"
```
Comment (vendethiel) [inline cooldowns_evoker.lua]: Why does this not have a cooldown?

File: cooldowns_evoker.lua [hunk 3]
```diff
@@ -0,0 +1,390 @@
...
+-- Emerald Blossom
+LCT_SpellData[355913] = {
+  class = "EVOKER",
+  cooldown = 30,
+  heal = true,
+}
+
+-- Return
+LCT_SpellData[361227] = {
+  class = "EVOKER"
```
Comment (vendethiel) [inline cooldowns_evoker.lua]: No cooldown either?

File: cooldowns_evoker.lua (hunk 2 above)
Comment (joshpc) [reply cooldowns_evoker.lua]: I blindly put all the spells (even those that are spammable.) Should I take them out?

Comment (vendethiel) [reply cooldowns_evoker.lua]: The only time I tend to do that is when that spell has special properties (for example, fireball reducing combustion cooldown, or spells reducing hammer of justice cooldown). And I add a `hidden = true` property so they’re not displayed in the options’ cooldown list.

**Pull request merged**